suite-common/test-utils/src/extraDependenciesMock.ts

```diff
@@ -70,12 +72,16 @@ export const extraDependenciesMock: ExtraDependencies = {
             showDebugMenu: false,
             transports: [],
         }),
+        selectMetadata: mockSelector('selectMetadata', {}),
+        selectDevice: mockSelector('selectDevice', {} as any),
```

Comment:
I don't think it's good idea to do `{} as any`, it could lead to some possible crashes in mobile app or failing tests, better use some mocked device object, we should have these for tests anyway.

Reply:
[28bfa47](https://github.com/trezor/trezor-suite/pull/9023/commits/28bfa478abe905c45e13cb926755f7f83fd4e136)

Reply:
There is already mock for tests for this. Good point.